Resources/Locale/en-US/job/role-timers.ftl

```diff
@@ -1,8 +1,8 @@
-﻿role-timer-department-insufficient = Require {TOSTRING($time, "0")} more minutes in {$department} department.
-role-timer-department-too-high = Requires {TOSTRING($time, "0")} fewer minutes in {$department} department. (Are you trying to play a trainee role?)
-role-timer-overall-insufficient = Require {TOSTRING($time, "0")} more minutes of playtime.
-role-timer-overall-too-high = Require {TOSTRING($time, "0")} fewer minutes of playtime. (Are you trying to play a trainee role?)
-role-timer-role-insufficient = Require {TOSTRING($time, "0")} more minutes with {$job} for this role.
-role-timer-role-too-high = Require {TOSTRING($time, "0")} fewer minutes with {$job} for this role. (Are you trying to play a trainee role?)
+﻿role-timer-department-insufficient = Requires {TOSTRING($time, "0")} more minutes in {$department} department for this role.
```

Comment:
What about "You require `{TOSTRING($time, "0")}` more minutes in `{$department}` department to play this role."?
Same for the other lines.